CHANGES.rst

```diff
@@ -1,6 +1,264 @@
 Traits CHANGELOG
 ================
 
+Release 6.3.0
+-------------
+
+Released: 2021-10-08
```

Comment:
This reflects the current plan for the release date; I'm not likely to find time to make the release before this Friday.